mediorum/main.go

```diff
@@ -157,6 +157,7 @@ func startDevCluster() {
 			Peers:             network,
 			Signers:           signers,
 			ReplicationFactor: 3,
+			PrivateKey:        getenvWithDefault(fmt.Sprintf("CN%d_SP_OWNER_PRIVATE_KEY", idx+1), "devkey"),
```

Comment:
I think the default should be nil since it expects it elsewhere to for dev. Or just update [this](https://github.com/AudiusProject/audius-protocol/blob/main/mediorum/server/server_basic_auth.go#L33) to expect "devKey" instead of nil